### ISSUE TITLE
Allure report "step name" is missing, if Cucumber step-name contains more than 80 characters

### ISSUE THREAD

**Issue by thomasKuntermannDiva-e** [mmisty/cypress-allure-adapter#108]
Hi @mmisty ,
it's me again :)

I'm facing an issue within the Allure report, if my Cucumber Step-name contains more than 80 characters.

Example:
```gherkin
Given Open baseUrl using mobile viewport
Then Verify the "visibility" of elementName: "sHeaderArea_brandLogo" from selectorFile: "selectors_homepage"
``` 

In this case, the "Given" step name is printed in Allure report, but the "Then" step name is empty:
![Bildschirmfoto 2024-03-21 um 14 46 58](https://github.com/mmisty/cypress-allure-adapter/assets/27360136/44598cb8-1096-401d-ba98-1ab3016d8d10)

The Allure result JSON file "name" attribute only contains the Cucumber buzzword "Then" in those cases:
<img width="408" alt="Bildschirmfoto 2024-03-21 um 14 52 27" src="https://github.com/mmisty/cypress-allure-adapter/assets/27360136/aab288ea-64e0-49b6-a6bb-a0d7c6888a67">

Any idea, how to resolve this?

Thanks in advanced and best regards

Thomas

**Comment by mmisty** [comment]
Hi @thomasKuntermannDiva-e , that's because of step length I assume if step length longer than 80 symbol it will not be shown(usaully it adds attachment with this step name)
This was done to avoid huge step names, I think I'm gonna increase that to something like 200 symbols

**Comment by thomasKuntermannDiva-e** [comment]
Ah yes, I can confirm this! I increased the Given statement w/o any string-literals to > 80 chars and then the step-name is not printed anymore.

I adjust the issue title accordingly and thanks again for your quick reply :)

**Comment by thomasKuntermannDiva-e** [comment]
Could this maybe also introduce a configurable env-variable?

**Comment by mmisty** [comment]
Hello! this was fixed in version 0.16.8
